openmdao/core/mpi_wrap.py

```diff
@@ -53,6 +53,7 @@ def under_mpirun():
         if name == 'OMPI_COMM_WORLD_RANK' or \
            name == 'MPIEXEC_HOSTNAME' or \
            name.startswith('MPIR_') or \
+           (name.startswith('MSMPI_') and name.startswith('PMI_RANK')) or \
```

Comment:
No because on windows, the MSMPI_ indicates that you have it installed.  The PMI_RANK is an environmental var that is set when you run under mpiexec.  Both conditions must be true to be running under MPI.